frontend/www/js/omegaup/components/arena/RunDetails.vue

```diff
@@ -213,9 +234,10 @@ export default class ArenaRunDetails extends Vue {
 
   EMPTY_FIELD = EMPTY_FIELD;
   T = T;
+  ui = ui;
   groupVisible: GroupVisibility = {};
   showFeedbackForm = false;
-  feedback = '';
+  feedback = this.data?.feedback ? this.data.feedback.feedback : '';
```

Comment:
```suggestion
  feedback = this.data?.feedback?.feedback ?? null;
```

es mejor usar `null` en vez de valores mágicos.